source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -28,5 +28,76 @@ more naturally readable.
 Operations
 ----------
 
+Each of the following sections provides information and examples for
+aggregation expression operations available in the {+driver-short+}.
+This guide sorts these operations into categories based on general
+purpose and functionality.
+
+.. important::
+
+   The Query API expression that the driver generates from each
+   aggregation expression operation might not be
+   identical to the Query API expression provided in each example.
+   However, they are functionally equivalent.
+
+Boolean Expressions
+~~~~~~~~~~~~~~~~~~~
+
+You can invoke boolean operations on boolean values of type ``MqlBoolean``.
+
+The following table describes the boolean aggregation operation methods
+available in the {+driver-short+} and the equivalent expressions in the
+Query API:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Aggregation Expression
+     - Query API Expression
+
+   * - `and() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlBoolean.html#and(com.mongodb.client.model.mql.MqlBoolean)>`__
+     - :manual:`$and </reference/operator/aggregation/and/>`
+       
+   * - `not() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlBoolean.html#not()>`__
+     - :manual:`$not </reference/operator/aggregation/not/>`
+
+   * - `or() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlBoolean.html#or(com.mongodb.client.model.mql.MqlBoolean)>`__
+     - :manual:`$or </reference/operator/aggregation/or/>`
+
+   * - `cond() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlBoolean.html#cond(T,T)>`__
+     - :manual:`$cond </reference/operator/aggregation/cond/>`
+
+.. note::
+
+   While each Java method is functionally equivalent to the
+   corresponding Query API expression, they may differ in expected
+   parameters and implementation.
+   
+The following code computes the ``result`` field based on if the value
+of ``field1`` is less than ``0`` *or* greater than ``10``:
```

Comment:
Suggestion:
I see this is similar to https://www.mongodb.com/docs/manual/reference/operator/aggregation/or/#example but not the same. However, I wonder if there might be conditions more familiar to a reader which could aid in understanding it more quickly.
Also, it could be useful to identify which operator this shows.

E.g. 
"The following example shows how to use the or() operator to ..."

Reply:
Would it be familiar to users to refer to a `qty` field with no introduction about what the document schema is?

Reply:
I wasn't specifically commenting on the field names, but I think using a more recognizable situation and field names might help. The current example is "Add a field called 'result' with the value of field1 if it's less than "0" or greater than "10". As far as I know, there isn't any natural association one might make as to why you would want to do that.

Not the most imaginative examples, but hopefully demonstrate what I mean:
1) if "name" equals "mouse" or "keyboard", set the value of the "type" field to "computer peripheral".
2) if "body_temperature" is less than "96" or "body_temperature" is greater than "100", set the value of the "action" to "see the doctor".

Reply:
Looks like you had something along the lines of what I was looking for in a prior revision about extreme temperatures which could still work. Was there any specific reason why you changed it other than that maybe it contained more than just the "or" operator?

Reply:
I just thought that following the discussion Mike, you, and I had, that we had agreed on generic field names. I have changed the example to something easily understood.

Reply:
Closing out since we discussed it synchronously and decided each example can either be:
1) generic field names when it's possible to clearly demonstrate/communicate its purpose without explanation
or
2) Provide a short explanation of what it does to help clarify what the aggregation does